### ISSUE TITLE
Gamma value frontend errors

### ISSUE THREAD

**Issue by Femkemilene** [cpmodel/FTT_StandAlone#108]
@pv-camecon, we are having some trouble calibrating the Gamma values for the FTT-P update through the frontend. The issue seems to be that there is not Output/Gamma.pickle file which the front end is trying to load in line 974 of FTT_backend.py. Could this be down to the fact that gamma output is also not being saved like the main ouput?

_Originally posted by @ldxib2 in https://github.com/cpmodel/FTT_StandAlone/issues/81#issuecomment-2018244196_

I tried determining gamma values using the REFEREE version on the SVN, and got a different error (it struggles with JSON files). I do remember this working, so will ask around what model version this worked.

**Comment by Femkemilene** [comment]
Cormac indicated it's just fickle and by pushing run_ftt multiple times, it does not produce errors anymore. The error produced is: 
![image](https://github.com/cpmodel/FTT_StandAlone/assets/26096675/76cb7b97-3bce-40d4-a6f6-bb43b75934c4)

When the gamma.pickle file is created and you restart the gamma tab in the front end, it does immediately work.

**Comment by pv-camecon** [comment]
Yes, the front-end will try to search for the Gamma.pickle file. Is this simply a matter a making sure that (for now) this pickle file is committed to the master branch?

**Comment by Femkemilene** [comment]
The gamma.pickle file is over 100 MB. We can technically add it, but it will make the repo much bigger. Do we know why it is required for the front-end?
